modules/ROOT/pages/push/coding.adoc

```diff
@@ -1,15 +1,134 @@
-= Registering Device
+include::{partialsdir}/attributes.adoc[]
 
-Use the following code to register a device with the {push-service} service:
+= Setting up {unifiedpush-service} service SDK
 
+This guide will help you to set up the {unifiedpush-service} service SDK in your App.
 
+== Prerequisites
 
 [role="primary"]
 .Android
 
 ****
+You need a Firebase account. See link:google-setup.html[Google setup] to get started.
+****
+
+[role="secondary"]
+.iOS
+
+****
+You need to set up APNs. See link:proc_preparing-ups-for-use-with-ios.adoc[Apple setup] to get started.
+****
+
+== Importing the libraries
+
+[role="primary"]
+.Android
+
+****
+
+. Add the push dependency to your application module
++
+[source,groovy]
+----
+implementation "org.aerogear:aerogear-push:[version]"
+implementation "com.google.firebase:firebase-messaging:[version]"
+----
++
+. Create a new project in link:https://console.firebase.google.com[Firebase console]
+. Add link:https://developers.google.com/android/guides/google-services-plugin[The Google Services Gradle Plugin] in your root `build.gradle` file
+[source,groovy]
+  buildscript {
+      dependencies {
+          classpath 'com.google.gms:google-services:[version]'
+      }
+  }
+. Apply the plugin in your app `build.gradle` file
+[source,groovy]
+  // ADD THIS AT THE BOTTOM
+  apply plugin: 'com.google.gms.google-services'
+. Download `google-services.json` an drop it in your app folder. For more details see link:google-setup.html[Google setup].
```

Comment:
drop => copy

Reply:
=> Download it in your app folder.
We should prevent the user from having the following Downloads folder:
```
google-services.json
google-services (2).json
google-services (3).json
google-services (4).json
```
![image](https://user-images.githubusercontent.com/11672286/41462887-a8cbed68-7094-11e8-88ee-4c878d783236.png)

Reply:
I'll remove this, it's already explained in the previous section and by that point you should have the app set up with that file.